docs/das/daserver-instructions.md

```diff
@@ -56,20 +48,29 @@ Copy the `das_bls` and `das_bls.pub` files to the <TODO SECRET VOLUME>
 
 ```
 /usr/local/bin/daserver --data-availability.mode local-disk --data-availability.local-disk.key-dir <TODO PERSISTENT VOLUME> --data-availability.local-disk.data-dir <TODO SECRET VOLUME>
---OPTION-NAME-TODO <SEQUENCER ADDRESS>
+--data-availability.store-signer <ETHEREUM ADDRESS>
 ```
 
-Alternatively the private key can be passed in via the command line:
+
+#### Running with private key on the command line
+```
+/usr/local/bin/datool keygen --dir /local/dir/of/your/choice
+```
+
+Copy the base64 encoded contents of the `das_bls` private key file.
+
 ```
 /usr/local/bin/daserver --data-availability.mode local-disk --data-availability.local-disk.priv-key "CONTENTS OF das_bls PRIVATE KEY FILE" --data-availability.local-disk.data-dir <TODO SECRET VOLUME>
---OPTION-NAME-TODO <SEQUENCER ADDRESS>
+--data-availability.store-signer <ETHEREUM ADDRESS>
 ```
 
 #### Passing in config by file
-Configuration can be passed in via the command line or via a JSON config file. Depending on your site setup it may be preferable to use a JSON configuration file. The JSON config for any `daserver` command line invocation can be generated by appending the `--conf.dump` flag and then once saved, `--conf.file` can be used to select the file.
+Configuration can be passed in via the command line or via a JSON config file. Depending on your site setup it may be preferable to use a JSON configuration file.
+
+The JSON config for any `daserver` command line invocation can be generated by appending the `--conf.dump` flag. This will dump all flags that you specified along with all other options with their default values. Before using the file it's recommended to remove the flags you are not using. The `--conf.file` flag can be used to select the file.
 
 ## Validating deployment
-In the docker image there is the `datool` utility that can be used to Store and Retrieve messages from a DAS. In order for the DAS to accept the Store requests, --OPTION-NAME-TODO must be ommitted when running `daserver` since you will not have the batch poster's private key to sign the messages with. --OPTION-NAME-TODO should be renabled when done verifying and before allowing external traffic.
+In the docker image there is the `datool` utility that can be used to Store and Retrieve messages from a DAS. In order for the DAS to accept the Store requests, --data-availability.store-signer must be set to 'none' when running `daserver` since you will not have the batch poster's private key to sign the messages with. The real Store signer address should be renabled when done verifying and before allowing external traffic.
```

Comment:
```suggestion
In the docker image there is the `datool` utility that can be used to Store and Retrieve messages from a DAS. In order for the DAS to accept the Store requests, --data-availability.store-signer must be set to 'none' when running `daserver` since you will not have the batch poster's private key to sign the messages with. The real Store signer address should be restored when done verifying and before allowing external traffic.
```